packages/embark/src/test/vm_fs.js

```diff
@@ -63,9 +65,12 @@ describe('fs', () => {
       it('should not throw exceptions on paths inside the temporary dir root', (done) => {
         assert.doesNotThrow(async () => {
           try {
+            if (func === 'readJSONSync') console.error = function() {};
```

Comment:
I'm assuming `readJSONSync` is writing to console.error in this case? Why?

Reply:
I was tired of seeing the output of `readJSONSync` in the test suite because it says `"error: ..."`, which is confusing because there wasn't actually a test error.

Reply:
readJSONSync is trying to read a file that doesn't actually exist, but the point of the test was to check that a permissions related exception wasn't thrown.